apps/central-scan/frontend/src/app_root.tsx

```diff
@@ -197,8 +198,8 @@ export function AppRoot({
   }, [setStatus]);
 
   const systemAdministratorUnconfigure = useCallback(() => {
-    unconfigureMutation.mutate({ ignoreBackupRequirement: true });
-  }, [unconfigureMutation]);
+    unconfigureMutate({ ignoreBackupRequirement: true });
+  }, [unconfigureMutate]);
```

Comment:
Couldn't this just be `unconfigure.mutate`?

Reply:
You would think, right? The `exhaustive-deps` rule does not allow it in this case.

Reply:
I think the reasoning is that, as a method, `mutate` may refer to `this` which needs to be captured in the dependency array. Which is why I think that sort of property-specific dependency is allowed for non-functions but not functions.

Reply:
https://github.com/facebook/react/issues/16265#issuecomment-517518539